README.md

```diff
@@ -123,20 +123,67 @@ Note: a single benchmark for FIO will take about 6 minutes to finish.
 
 See [./deploy/fio.yaml](https://github.com/yasker/benchmark/blob/main/deploy/fio.yaml) for available options.
 
-#### Deploy Comparison Benchmark in Kubernetes cluster
+### Deploy Single Volume Benchmark via Helm
+
+All defaults are the same as in the YAML installation method.
+Use the following steps to prep for installation:
+```bash
+# add helm repo
+helm repo add yasker-benchmark https://raw.githubusercontent.com/yasker/benchmark/main/helm
+
+# create namespace for needed resources
+kubectl create ns fio-benchmarks
+```
+
+To deploy with all defaults:
+```bash
+helm upgrade --install fio-benchmark-default \
+  yasker-benchmark/fio-standalone \
```

Comment:
We can just call it `fio`. I think we should be able to cover both single and compare cases in the chart. Comparison is very useful, don't want to just leave it to yaml, lol.

Reply:
My thought was we could move a comparison benchmark into "install the helm chart multiple times" to allow N comparisons easily - thoughts for maybe reworking the README to have a YAML-based comparison and a Helm-based comparison?